apps/onboarding-functions/src/main/resources/application.properties

```diff
@@ -161,3 +161,4 @@ notification.consumers.fd.name=${FD_SHARED_ACCESS_KEY_NAME:test}
 notification.consumers.fd.key=${EVENTHUB_SC_CONTRACTS_FD_SELFCARE_WO_KEY_LC:test}
 notification.consumers.fd.allowed-institution-types=null
 notification.consumers.fd.allowed-origins=null
+notification.minutes-threshold-for-update-notification=${MINUTES_THRESHOLD_FOR_UPDATE_NOTIFICATION:5}
```

Comment:
ditto, how is it used?

Reply:
The property represents a threshold expressed in minutes that is used to check whether an onboarding is to be considered as "JUST COMPLETED" or "UPDATED." The check is performed by verifying that the difference between updatedAt and createdAt does not exceed the threshold. If it exceeds it we consider that updating the record on onboarding should send an UPDATE event to the topics. If, on the other hand, the threshold is not exceeded we believe we should send an ADD event.